app/src/main/java/com/codelab/android/datastore/data/UserPreferencesRepository.kt

```diff
@@ -48,12 +48,11 @@ data class UserPreferences(
 /**
  * Class that handles saving and retrieving user preferences
  */
-class UserPreferencesRepository private constructor(context: Context) {
+class UserPreferencesRepository (private val context: Context) {
 
     private val TAG: String = "UserPreferencesRepo"
 
-    private val dataStore: DataStore<Preferences> =
-        context.createDataStore(
+    private val Context.dataStore: DataStore<Preferences> by preferencesDataStore(
```

Comment:
Ah someone brought this up in a bug report already, but we actually want this at the top level outside of the UserPreferencesRepository class... but we can't create a SP migration there